Src/BootCamp.Chapter/Settings.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public static class Settings
+    {
+        public static NumberFormatInfo numberFormatInfo = new NumberFormatInfo { NumberDecimalSeparator = "." };
+        public static CultureInfo cultureInfo = new CultureInfo("en-GB");
+        public static char divider = ',';
+        public static string currency = "\u00A4";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'currency' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5054323)